wallet_presentation/src/commonMain/kotlin/net/thechance/mena/wallet/presentation/screen/transaction_history/TransactionHistoryViewModel.kt

```diff
@@ -32,7 +33,8 @@ import kotlin.uuid.Uuid
 @OptIn(ExperimentalUuidApi::class)
 @KoinViewModel
 class TransactionHistoryViewModel(
-    @Provided private val transactionRepository: TransactionRepository
+    @Provided private val transactionRepository: TransactionRepository,
+    private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO
```

Comment:
```suggestion
    private val dispatcher: CoroutineDispatcher = Dispatchers.IO
```

I think it's better to name it just dispatcher because we change it in the test to testDispatcher so it's not always ioDispatcher

Reply:
@Joseph-Sameh-0  yah its sound good but here i rename as ioDispatcher cuz in any ViewModel here we pass it as io dispatcher, and in test as usual we rename it test dispatcher